test/modellib/GRUModel.cpp

```diff
@@ -127,7 +127,7 @@ bool GRULibBuilder::prepareInputs(float dataRangeLB, float dataRangeUB) {
       llvm::ArrayRef(xShape), dataRangeLB, dataRangeUB);
   list[1] = omTensorCreateWithRandomData<float>(
       llvm::ArrayRef(hShape), dataRangeLB, dataRangeUB);
-  inputs = omTensorListCreateWithOwnership(list, num, true);
+  inputs = omTensorListCreate(list, num);
```

Comment:
same issue, list alloc on line 123 is not freed anymore. Please use static array declared on the stack.

Reply:
will do

Reply:
Okay seems like all of the files in `test/modellib/` needed to be updated. Hopefully I caught them all.

Reply:
It was an oversight… this is updated now